src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = reader.readLine();
+            while (value != null) {
+                if (value.startsWith("supply,")) {
+                    String supplyResult = value.substring(value.indexOf(',') + 1);
+                    supply += Integer.valueOf(supplyResult);
+                } else {
+                    String buyResult = value.substring(value.indexOf(',') + 1);
+                    buy += Integer.valueOf(buyResult);
```

Comment:
Instead of using Integer.valueOf(), you could use Integer.parseInt(). The former returns a new Integer() object, whereas the latter returns a primitive int.